content/raw/chat/online_users.md

```diff
@@ -61,10 +68,13 @@ preconditions:
 
 **Status Selector**
 
-- The user MUST be able to select his state as 'online' or 'offline'
-- If the user has 'online' as selected, then they MUST be shown online in the communities member list.
-- If the user has 'offline' as selected, then they MUST be shown offline for everyone within 5 minutes regardless of activity, even if the user sends new messages.
+- The user MUST be able to select his state as 'always online' or 'inactive' or 'set status automatically'
```

Comment:
'set status automatically' should be the default setting for new users.  What status the user has set in this menu should persist through app restarts.